src/Components/Home/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "../ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-app-ftk2.onrender.com/api/articles")
```

Comment:
You may want to extract fetching behaviour here to utils or api file. you'll have a lot of functions like this. Please use axios.create() too so you will have a base URL which can be used for all your request functions. 💡